test/ssl-tests/14-curves.cnf.in

```diff
@@ -10,14 +10,26 @@ use warnings;
 use OpenSSL::Test;
 use OpenSSL::Test::Utils qw(anydisabled);
 
-my @curves = ("sect163k1", "sect163r1", "sect163r2", "sect193r1",
-              "sect193r2", "sect233k1", "sect233r1", "sect239k1",
-              "sect283k1", "sect283r1", "sect409k1", "sect409r1",
-              "sect571k1", "sect571r1", "secp160k1", "secp160r1",
-              "secp160r2", "secp192k1", "prime192v1", "secp224k1",
-              "secp224r1", "secp256k1", "prime256v1", "secp384r1",
-              "secp521r1", "brainpoolP256r1", "brainpoolP384r1",
-              "brainpoolP512r1", "X25519", "X448");
+our $fips_mode;
+
+my @curves;
+
+if ($fips_mode) {
```

Comment:
Would it make sense to start with the fips list and add the non-fips if `!$fips_mode`?

This avoids future updates that miss one of the if branches.

Reply:
Yes that makes sense. Fix coming up...